org.eclipse.jdt.ls.core/logback.xml

```diff
@@ -1,10 +1,3 @@
 <configuration scan="true">
-  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-    <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-      <pattern>%date [%thread] %-5level %logger{35} - %msg%n</pattern>
-    </encoder>
-  </appender>
-  <root level="DEBUG">
-    <appender-ref ref="STDOUT" />
-  </root>
```

Comment:
Was there a reason to remove this ? This configuration file is what gets read and written out below, and then set to ensure m2e uses it for logging. Without this I don't think any logging will happen of m2e.

Reply:
We can't use logback's ConsoleAdapter. It breaks Java LS stdout. We can set some FILE adapter.